NOTES-WINDOWS.md

```diff
@@ -138,8 +138,8 @@ Native builds using Embarcadero C++Builder
 =========================================
 
 This toolchain (a descendant of Turbo/Borland C++) is an alternative to MSVC.
-OpenSSL currently includes an experimental 32-bit configuration targeting the
-Clang-based compiler (`bcc32c.exe`) in v10.3.3 Community Edition.
+OpenSSL currently includes an experimental 32-bit and 64-bit configuration targeting the
```

Comment:
"an" is singular, but we now have two configurations, so 
```suggestion
OpenSSL currently includes experimental 32-bit and 64-bit configurations targeting the
```

Reply:
fix it here https://github.com/openssl/openssl/pull/23377/commits/ed9e258e2e10dad91e43bbc342b73d35d66b5591